src/components/DisplayComponent.js

```diff
@@ -14,6 +24,33 @@ class DisplayComponent extends Component {
   }
 
   render() {
+    var display;
+    if (this.state.displayComponent === 'Chart') {
+      display = (
```

Comment:
This if/else stmt is getting large, can you break it down into smaller helper components say `<Chart />` and `<Map />` components?

Reply:
Good point! Will do that. Thank you